app/code/Magento/VaultGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,51 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Query {
+    paymentTokenList: [PaymentToken] @resolver(class: "Magento\\VaultGraphQl\\Model\\Resolver\\PaymentTokenList") @doc(description: "The paymentTokenList query returns a list of stored payment methods for a logged-in customer")
+}
+
+type Mutation {
+    paymentTokenAdd(input: PaymentTokenInput!): PaymentToken @resolver(class: "Magento\\VaultGraphQl\\Model\\Resolver\\PaymentTokenAdd") @doc(description: "Add payment tokens")
+    paymentTokenDelete(id: Int!): Boolean @resolver(class: "Magento\\VaultGraphQl\\Model\\Resolver\\PaymentTokenDelete") @doc(description: "Remove stored payment tokens")
+}
+
+type PaymentToken @doc(description: "PaymentToken defines the stored payment details") {
+    entity_id: Int @doc(description: "The ID assigned to the payment token")
+    customer_id: Int @doc(description: "The customer ID")
```

Comment:
No need to return `cutsomer_id` because customer can only query their own tokens